### PR TITLE
Add classic support for WowAce

### LINKED ISSUE #32: Add configuration and support for classic addons from all sources
- [x] Curse - #46 
- [ ] WoWAce
- [x] WoWInterface - #64

### PR DESCRIPTION
**Pull request by PeachIceTea**
This adds classic support for WowAce. It also fixes a bug caused by a wrong assumption that the first file in the files list is always a retail version. 

This closes #32

### CODE REVIEW

**Comment by grrttedwards** [comment]
Hey! This is awesome! I like taking the initiative to ditch the nasty regex HTML parsing 😛 
Hopefully some of that greatness can spill into the Curse code sooner or later.

**Comment by grrttedwards** [comment]
Two things:
1. It'd be great to have the tests updated not just to pass, but also to test the new retail/classic code. See `test/site/test_curse.py` which got a bit messy, but tests both the retail and classic modes using `unittest`'s subtest feature.
1. Could you update the version to `v1.4.0` in `changelog.txt` and write a sentence explaining this change?

**Comment by PeachIceTea** [comment]
Sorry for the force pushes. I am what you would call an amateur. I hope this is OK otherwise.

File: test/site/test_curse.py [hunk 1]
```diff
@@ -33,8 +33,6 @@ def test_integration_curse_get_latest_version(self):
         for url, version_regex in version_test_data:
             for game_version in GameVersion.__members__.values():
                 with self.subTest((game_version, url, version_regex)):
-                    if 'weakauras' in url and game_version is GameVersion.classic:
-                        return  # weakauras doesn't have a classic download link yet
```
Comment (grrttedwards) [inline test/site/test_curse.py]: +1 Thank you for checking and updating this :)

**Comment by grrttedwards** [comment]
No worries on the force pushes, it's on your own fork anyways :)

Looks good, I think! Thank you!!

**Pull request merged**